src/generators/clientFileGenerator.ts

```diff
@@ -26,6 +26,11 @@ export function generateClient(clientDetails: ClientDetails, project: Project) {
     }
   );
 
+  clientFile.addImportDeclaration({
```

Comment:
What is this import needed for? Looking at the generated files it doesn't look like `coreHttp` is being used where this is included.

Reply:
Hmm not needed I think I was experimenting with something and didn't remove. I'll take this out